src/unit.js

```diff
@@ -0,0 +1,38 @@
+define([
+	"./core",
+	"./unit/format",
+
+  "./plural"
+], function( Globalize, unitFormat ) {
+
+/**
+ * Globalize.formatUnit( value, unit, options )
+ *
+ * @value [Number]
+ *
+ * @unit [String]:
+ *
+ * @options [Object]
+ * - form: [String] "long", "short" (default), or "narrow".
+ *
+ * Format units such as seconds, minutes, days, weeks, etc.
+ */
+Globalize.prototype.formatUnit = function( value, unit, options ) {
+  var pluralGenerator;
+
+	if ( typeof value !== "number" ) {
+		throw new Error( "Value is not a number" );
+	}
+
+	if ( !unit ) {
+		throw new Error( "Missing unit" );
+	}
+
+	pluralGenerator = this.pluralGenerator();
+
+	return unitFormat( value, unit, options, pluralGenerator, this.cldr, this );
+};
```

Comment:
In general, `.formatUnit()` should be an alias for `.unitFormatter()`. Like `.formatNumber()` https://github.com/jquery/globalize/blob/master/src/number.js#L148-L154 is for `.numberFormatter()` https://github.com/jquery/globalize/blob/master/src/number.js#L107-L137

Note the process of setting up the formatter in general is somewhat an expensive operation. This process includes traversing CLDR data and cherry-picking the specific data (which I name `properties`) for the particular options used by the formatter. Therefore, the formatter generator should return a function that in turn allows to execute the formatting (i.e., allow user to setup once and execute several times for optimal performance; and to allow runtime compiled formatters). Please, just let me know on any questions.